app/Resources/views/autoEdits/result.html.twig

```diff
@@ -0,0 +1,68 @@
+{% extends 'base.html.twig' %}
+
+{% block body %}
+<div class="col-lg-12">
+<div id="wrapper">
+<main id="content" class="clearfix">
+        <div class="col-lg-8 col-lg-offset-2">
+            </div>
+    
+    <div class="panel panel-primary">
+    <header class="panel-heading">
+        <div class="text-center xt-heading-top">
+            <a class="back-to-search" href="{{ path("autoedits") }}"">
+                <span class="glyphicon glyphicon-chevron-left"></span>
+                Back
+            </a>
+            {{ wiki_link("User:" ~ username, url, username) }} &middot; {{ wikiName }}
+        </div>
+    </header>
+    <div class="panel-body xt-panel-body">
+    		<p>
+				<a href="{{ linkWikiScript(url, "title=Special:Log&type=block&user=&year=&month=-1&tagfilter=&user=User:"~username) }} ">{{ msg("block_log") }}</a>
+				{% if isWMFLabs() %}
+				&middot;
+				<a href="//tools.wmflabs.org/guc/?user={{username}}" >Global user contributions</a>
+				&middot;
+				<a href="//meta.wikimedia.org/w/index.php?title=Special%3ACentralAuth&target={{username}}" >Global Account Manager</a>
+				{% endif %}
+			</p>
+
+			<div class="panel panel-default">
+				<div class="panel-heading">
+					<h4  class="topcaption" >Summary <span class="showhide" onclick="javascript:switchShow( 'summary', this )">[hide]</span></h4>
```

Comment:
No inline JavaScript, ever! :) The new system to do this is to use the `macros/layout.html.twig` helpers and let `application.js` do it's thing.

Check out https://github.com/x-tools/xtools-rebirth/blob/master/app/Resources/views/articleInfo/result.html.twig#L594-L628 on how to make a block using the layout helpers. The general format is:
```
{% set content %}
  Your inner content, no .panel-body wrappers or anything
{% endset %}
{{ layout.content_block('i18n-message', content) }}
```

So here `i18n-message` would be `summary`